src/allotropy/parsers/unchained_labs_lunatic/unchained_labs_lunatic_reader.py

```diff
@@ -0,0 +1,11 @@
+from io import IOBase
+
+import numpy as np
+import pandas as pd
+
+
+class UnchainedLabsLunaticReader:
+    def __init__(self, contents: IOBase):
```

Comment:
an `__init__` method should only validate parameters and assign them to fields. Please move work like calling `pd.read_csv` to an instance method.